docs/siem/detections/detection-engine-intro.asciidoc

```diff
@@ -29,106 +62,44 @@ To make sure you can access Detections and manage rules, see
 <<detections-permissions>>.
 ==============
 
-[role="screenshot"]
-image::detections-ui.png[]
-
 [float]
 [[det-engine-terminology]]
 == Terminology
 
-Rules::
-Perform background tasks to detect suspicious activity. There are two types of
-rules:
-
-* Query-based rules search indices for documents that match their queries at
-scheduled intervals. When a document matches, a signal is created.
-* {ml-cap} rules create a signal when a {ml} job discovers an anomaly score
-above a rule's defined threshold.
+Actions::
+Sends notifications via other systems when a detection alert is created, such
+as email, Slack, PagerDuty, and Webhook.
 
-Signals::
-Always refer to {siem-soln} produced detections. Signals are never received 
-from external systems. When a rule's conditions are met, the {siem-app} 
-writes one or more signals to an Elasticsearch `signals` index.
+Detection alerts::
+{es-sec} produced alerts. Detection alerts are never received from external
+systems. When a rule's conditions are met, {es-sec} writes a detection alert to
+an Elasticsearch `.signals` index.
 +
 [NOTE]
 ==============
-Signal indices are created for each {kib} space. The naming convention is:
-`.siem-signals-<space name>`. For the default space, the signals index is named 
-`.siem-signals-default`.
+Detection alert indices are created for each {kib} space. The naming convention
+is: `.siem-signals-<space name>`. For the default space, the alerts index is
+named `.siem-signals-default`.
 ==============
 
-Alerts and events::
-Always refer to data the {siem-app} receives from external systems, such as 
-Elastic Endpoint and Suricata.
+Detection rules::
+Background tasks that run periodically and produce alerts when suspicious
+activity is detected.
 
-Actions::
-Used to send notifications via other systems when a signal is created, such as
-email, Slack, PagerDuty, and Webhook.
-
-[float]
-== Signals and external alerts
+Endpoint exceptions::
+<<term-exceptions, Exceptions>> added to both rules and Endpoint agents on
+hosts. Endpoint exceptions can only be added when:
 
-The Detections page displays all signals and alerts. To view signals created 
-by a rule, you can:
+* Endpoint agents are installed on the hosts.
+* The Elastic Endpoint Security rule is activated.
```

Comment:
So the two constraints for being able to add endpoint exceptions are as follows:

1. The Detection Rule has the `Associate rule to Global Endpoint Exception List` setting checked from the `About` section of the Rule. In which case the option to `Add Endpoint exception` will be available from the Rule Details page.

-or-

2. The Detection Alert generated from a rule has the following fields: 
`signal.original_event.module determined:endpoint and signal.original_event.kind:alert `, in which case the `Add endpoint exception` action [will be available](https://github.com/spong/kibana/blob/c21474b4ce029b820072b53f3908075404bccbde/x-pack/plugins/security_solution/public/detections/components/alerts_table/default_config.tsx#L310-L320), and if used will end up associating the rule that generated the alert with the `Global Endpoint Exception List`. 

We don't have copy for this latter part in the app, so we should call this out somewhere in the documentation (not sure if here works, but might fit well?).